model/registry/event.yaml

```
@@ -16,3 +16,13 @@ groups:
           separation of semantics for events in separate domains like browser, mobile, and
           kubernetes.
         examples: ['browser.mouse.click', 'device.app.lifecycle']
+      - id: description
+        type: string
+        stability: experimental
+        brief: >
+          Human-readable message representing an event.
```

```suggestion
          Human-readable description representing an event.
```
another missed rename I think?